app/src/main/java/wishcantw/vocabulazy/activities/player/view/PlayerOptionView.java

```diff
@@ -154,9 +152,9 @@ public void setOptionInModeContent(OptionSettings option, boolean init) {
         mListOrderRadioGroup.check(listOrderIdx);
         mVocOrderRadioGroup.check(vocOrderIdx);
 
-        mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_REPEAT, option.getItemLoop());
+        mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_REPEAT, option.getItemLoop()-1);
         mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_SPEED, option.getSpeed());
-        mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_PLAY_TIME, option.getPlayTime());
+        mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_PLAY_TIME, option.getPlayTime()-10);
```

Comment:
The purpose of the deduction is standardization.

For item loop, the seek bar requires value 0~4, and since the value of item loop in option setting is 1~5, we need to minus 1 for standardization.

Similar as item loop, play time requires 0~30 in seek bar but will get 10~40 in option settings, therefore the deduction of 10 will standardize the value.